src/olympia/core/languages.py

```diff
@@ -209,7 +209,7 @@
         'native': '\u0938\u0902\u0938\u094d\u0915\u0943\u0924',
     },
     'sah': {'english': 'Sakha', 'native': '\u0421\u0430\u0445\u0430\u043b\u044b\u044b'},
-    'sat': {'english': 'Santali', 'native': '\u0938\u0902\u0924\u093e\u0932\u0940'},
+    'sat': {'english': 'Santali', 'native': '\u1c65\u1c5f\u1c71\u1c5b\u1c5f\u1c72\u1c64'},
```

Comment:
```suggestion
    'sat': {
        'english': 'Santali',
        'native': '\u1c65\u1c5f\u1c71\u1c5b\u1c5f\u1c72\u1c64',
    },
```

to satisfy `black` (otherwise you can run `black` yourself on that file)